### PR TITLE
[WIP] Standup2 changes

### PR DESCRIPTION
**Pull request by willkg**
This adds some docker shenanigans to make it easier for me to run tests without installing node and friends on my system because I'm old. Having said that, the docker stuff is not helpful for anything other than running tests.

This also strips out all the commands that relied upon the /api/v2/ stuff in standup as well as the /api/v2/ enabling code.

This also changes the user settings update command so that users can only adjust their own settings--no more admin adjusting user settings. Admin can do that on the website now.

r?

### CODE REVIEW

**Comment by willkg** [comment]
Making this WIP for now because we can't land this until we nix the autodeploy stuff.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+FROM node:4.2.4
+
+WORKDIR /app
+
+# Install node requirements and clean up unneeded cache data
+COPY package.json package.json
+RUN cd /app && npm install -d
```
Comment (pmac) [inline Dockerfile]: shouldn't need the `cd /app` bit since WORKDIR above sets that for you.

**Comment by pmac** [comment]
Looks like a good change to me. Do we want to bring the team management stuff back later, or move that to the web app? I feel like we've said the latter before. If I'm remembering correctly then r+. Only other thing I'd say is that maybe instead of completely removing commands we should just have those old commands say someting to the calling user like "I can no longer manage teams. Bother the admins."

**Comment by willkg** [comment]
So... I'm working off the theory that only a few people have ever used the team commands and they've used them rarely. Given that, I don't feel compelled to elaborate for specific things like this.

We could add more data to the `!help` output and say something like, "For everything else, either log into the Stanup web app or view the wiki page at xyz for how to do it." That could be a nice catch-all for things.

Does that sound helpful?

**Comment by pmac** [comment]
Sounds great. I'm sure your theory is sound.

**Comment by willkg** [comment]
^^^ That fixes the Dockerfile and adds a blurb to the `!help`. I think that covers everything.

**Comment by willkg** [comment]
^^^ That un-hardcodes the text.

**Comment by willkg** [comment]
^^^ That is a force push after rebasing against the current master tip with the https changes.

**Comment by pmac** [comment]
The failure is jshint complaining about you using `config['standup']['website']` instead of `config.standup.website`.

**Comment by willkg** [comment]
Bah. Curse my fading JS-fu.

**Comment by pmac** [comment]
w00t. r+ from me. want to squash or just merge?

**Pull request merged**